source/nodejs/adaptivecards-ui-testapp/src/rendering-utils.ts

```diff
@@ -129,7 +135,8 @@ export function renderCard(cardJson: any, callbackFunction: Function): void {
 
 export function cardRenderedCallback(renderedCard: HTMLElement) {
     const renderedCardDiv = document.getElementById("renderedCardSpace");
-    renderedCardDiv.innerHTML = "";
+    const trustedHtml = ttPolicy?.createHTML("") ?? "";
+    renderedCardDiv.innerHTML = trustedHtml as string;
```

Comment:
as I note in `card-elements.ts` below, there's an `emptyHTML` member we can use:

```typescript
function clearElement(element: HTMLElement) : void {
    const trustedHtml = window.trustedTypes?.emptyHTML ?? "";
    element.innerHTML = trustedHtml as string;
}
```

Reply:
resolved